src/toast.tsx

```diff
@@ -209,10 +209,14 @@ const Toast: FC<ToastProps> = (props) => {
 
   const getPanResponder = () => {
     if (panResponderRef.current) return panResponderRef.current;
+    const swipeThreshold = Platform.OS === "android" ? 10 : 0;
     panResponderRef.current = PanResponder.create({
       onMoveShouldSetPanResponder: (_, gestureState) => {
         //return true if user is swiping, return false if it's a single click
-        return !(gestureState.dx === 0 && gestureState.dy === 0);
+        return (
+          Math.abs(gestureState.dx) > swipeThreshold ||
+          Math.abs(gestureState.dy) > swipeThreshold
+        );
```

Comment:
nit but maybe following is slightly preferable:

```typescript
PanResponder.create({
  onMoveShouldSetPanResponder: Platform.select({
    default: () => true,
    android: (e: GestureResponderEvent, state: PanResponderGestureState) =>
      Math.abs(state.dx) > swipeThreshold || Math.abs(state.dy) > swipeThreshold
  })
})
```

Reply:
Hi @alyosha, the value of `swipeThreshold` in the fix should limit the behavior change to Android only.

In your suggestion, every platform would change behavior in the case where both `state.dx` and `state.dy` are `0` (i.e. it would change the return value of `onMoveShouldSetPanResponder` from `false` to `true`). That might work fine too, but it seems risky to me (I would assume the current behavior of letting the return value depend on whether `dx` and `dy` are both `0` is there for a reason).

Reply:
oops -- definitely agree safer to maintain existing behavior for non-Android platforms, and testing on my side I did notice behavioral change/issue on iOS with my above code 😅 🤦 

so maybe just keep existing logic for default case? 

```typescript
onMoveShouldSetPanResponder: Platform.select({
  default: (_: GestureResponderEvent, state: PanResponderGestureState) =>
  !(state.dx === 0 && state.dy === 0),
  android: (_: GestureResponderEvent, state: PanResponderGestureState) =>
    Math.abs(state.dx) > swipeThresholdAndroid ||
    Math.abs(state.dy) > swipeThresholdAndroid,
}),
```

Reply:
Correct me if I'm wrong @alyosha but I think my proposed fix is functionally equivalent to your suggestion. The special behavior for Android is handled in the conditional declaration of `swipeThreshold`.